lib/plugins/EPrints/Plugin/Import/PubRouter.pm

```diff
@@ -495,92 +454,24 @@ sub xml_to_epdata
 		}
 	}
 
-	#free_to_read
-	my $free_to_read = $xml->getElementsByTagNameNS( $namespaces{ali}, "free_to_read" )->item(0);
-	if( $free_to_read && $dataset->has_field( "rioxx2_free_to_read_input" ) )
-	{
-		my $start_date = $free_to_read->getAttribute( "start_date" );
-		my $end_date = $free_to_read->getAttribute( "end_date" ) ;
-		$epdata->{rioxx2_free_to_read_input} = {
-			free_to_read => "Yes",
-			start_date => $start_date,
-			end_date => $end_date,
-		};
-	}
-
 	#license
-	my @licenses = $xml->getElementsByTagNameNS( $namespaces{pr}, "license" );
-	
-	#create the additional information content
-	my %license_notes;
-	my %licenses;
-	foreach my $license (@licenses)
+	my $license = $xml->getElementsByTagNameNS( $namespaces{ali}, "license_ref_start_date" )->item(0);
```

Comment:
Is it right that it says "license_ref_start_date" or should that be simply "license_ref"?

Reply:
Bug fixed with https://github.com/jisc-services/eprints-plugin-PubRouter-RIOXXplus-Connector/pull/5/commits/5d2140888ed57771e75f7fe23225da064962ed62